plugins/tekton/src/components/common/Status.tsx

```diff
@@ -1,78 +1,93 @@
 import React from 'react';
 
 import {
+  AngleDoubleRightIcon,
   BanIcon,
+  CheckCircleIcon,
   ClipboardListIcon,
+  ExclamationCircleIcon,
+  ExclamationTriangleIcon,
   HourglassHalfIcon,
   HourglassStartIcon,
   NotStartedIcon,
   SyncAltIcon,
   UnknownIcon,
 } from '@patternfly/react-icons';
+import classNames from 'classnames';
 
 import { StatusIconAndText } from '@janus-idp/shared-react';
 
-import {
-  GreenCheckCircleIcon,
-  RedExclamationCircleIcon,
-  YellowExclamationTriangleIcon,
-} from './icons';
+import './Status.css';
 
-export type StatusComponentProps = {
-  title?: string;
+export type StatusProps = {
   iconOnly?: boolean;
-  noTooltip?: boolean;
   className?: string;
-  popoverTitle?: string;
-};
-
-export type StatusProps = StatusComponentProps & {
   status: string;
+  height?: number;
+  width?: number;
+  spin?: boolean;
 };
 
 const DASH = '-';
 
 /**
  * Component for displaying a status message
  * @param {string} status - type of status to be displayed
- * @param {string} [title] - (optional) status text
  * @param {boolean} [iconOnly] - (optional) if true, only displays icon
- * @param {boolean} [noTooltip] - (optional) if true, tooltip won't be displayed
  * @param {string} [className] - (optional) additional class name for the component
- * @param {string} [popoverTitle] - (optional) title for popover
  * @param {ReactNode} [children] - (optional) children for the component
+ * @param {number} [height] - (optional) height of the status icon
+ * @param {number} [width] - (optional) width of the status icon
+ * @param {boolean} [spin] - (optional)
+
  * @example
  * ```tsx
  * <Status status='Warning' />
  * ```
  */
 export const Status = ({
   status,
-  title,
   iconOnly,
-  noTooltip,
   className,
-}: React.PropsWithChildren<StatusProps>) => {
+  height,
+  width,
+  spin,
+}: React.PropsWithChildren<StatusProps>): React.ReactElement => {
   const statusProps = {
-    title: title || status,
+    title: status ?? '',
```

Comment:
This looks confusing, why did we remove the title prop and pass just the status to the title?

Reply:
Because we are not passing any value to the title prop anywhere when calling Status